serverless/serverless
# CloudFormation normalization for WebSockets deployment resource

## What did you implement


Closes https://github.com/serverless/serverless/issues/5940

## How can we verify it
1. Create a service that creates WebSockets resources. For example, you may use [this example project](https://github.com/serverless/examples/tree/master/aws-node-websockets-authorizers).
2. Run `sls deploy`.
3. Run `sls deploy` once again. The expected result provided by this PR is that instead of redeployment, the deployment should be skipped. You should see `Service files not changed. Skipping deployment...` message.


## Todos

<details>
<summary>Useful Scripts</summary>


- `npm run test:ci` --> Run all validation checks on proposed changes
- `npm run lint:updated` --> Lint all the updated files
- `npm run lint:fix` --> Automatically fix lint problems (if possible)
- `npm run prettier-check:updated` --> Check if updated files adhere to Prettier config
- `npm run prettify:updated` --> Prettify all the updated files

</details>


**_Is this ready for review?:_** YES
**_Is it a breaking change?:_** NO
